packages/core/src/components/card/card.scss

```diff
@@ -11,15 +12,17 @@
   display: block;
 }
 
-:host(:hover) {
-  .dso-card-heading a {
-    color: card.$heading-anchor-hover-color;
-    text-decoration: underline;
-  }
+:host(:focus-within) {
+  background-color: colors.$grijs-5;
 }
 
 .dso-card-heading a {
   text-decoration: none;
+
+  &:hover {
+    color: card.$heading-anchor-hover-color;
```

Comment:
Bovenstaande opmerking over de semantische (specifieke) variable name geldt ook voor deze.

Reply:
Is `$heading-anchor-hover-color` dan niet semantisch?